<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="condBtn">
   <property name="geometry">
    <rect>
     <x>582</x>
     <y>20</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>условие задачи</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buildBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>построить деревья и таблицы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="strEdt">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>строка</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>69</y>
     <width>711</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>деревья</string>
    </attribute>
    <widget class="QLabel" name="avlLbl">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>321</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>avlTree</string>
     </property>
    </widget>
    <widget class="QLabel" name="btreeLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>321</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>btree</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>бинарное дерево</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>авл-дерево</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>таблицы</string>
    </attribute>
    <widget class="QTableWidget" name="openTbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>361</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="rowCount">
      <number>41</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QTableWidget" name="closeTbl">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>131</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="rowCount">
      <number>41</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="testBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>тесты текущего состояния структур</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>буква</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="chEdt">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>58</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="findBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>56</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>найти</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
